Header header
uint32 node_id

float64[] expected_measurement
float64[] pixel_measurement